main.yaml

```diff
@@ -1,48 +1,41 @@
+      structure of gas giant planets. Addressing the entire phase diagram with accurate 
+      ab initio simulations like diffusion Monte Carlo (DMC) is not currently feasible 
+      due to the computational cost, which limits studies to small system sizes. 
+      Recently, machine-learned interatomic potentials trained on ab initio data have 
+      been applied in large-scale molecular dynamics simulations to approach the accuracy 
+      of the ab initio methods without the finite size errors. Typically, these have relied 
+      on density functional theory to generate the training data. Here we provide the a
+      large-scale DMC database for dense hydrogen, which allows training of machine-learned 
+      potentials with DMC accuracy.
 
-      <p>A beta version of the system is available at </p>
+    image: img/f10e_grid-phase-diagram.pdf
 
-      The platform has been designed based on community input
-      primarily through working groups and collaborations with researchers.
-    image: img/wt01.svg
-  - title: Why QMC-HAMM?
+  - title: Stable solid molecular hydrogen above 900K
     name: why-wt
     content: |
-      Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium
-      doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo
-      inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.
-      Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut
-      fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem
-      sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit
-      amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora
-      incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad
-      minima veniam, quis nostrum exercitationem ullam corporis suscipit
-      laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum
-      iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae
-      consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?
-    image: img/wt02.svg
-  - title: Third Card
+      Recently we used a new potential trained on this DMC data in large-scale path 
+      integral molecular dynamics simulations to study molecular hydrogen. We found 
+      a phase diagram with HCP and C2/c-24 phases and two new structures with Fmmm-4 
+      molecular centers. The Fmmm-4 structures show a molecular orientational order 
+      transition from an ordered low-temperature structure to an isotropic high-temperature 
+      phase which melts to a molecular liquid with a maximum melting temperature of 
+      1450K at 150 GPa. This finding will likely lead to new experimental studies 
+      of the melting curve for molecular hydrogen.
+
+    image: img/f70a_solid-phases.pdf
+  - title: yt Hub 
+    name: what-tale
+    content: |
+    image: img/ythub.png
+
+  - title: How to use the database
     name: what-tale
     content: |
```

Comment:
I think you might want to leave this empty, rather than the `|`, which might confuse the parser.

Reply:
I am okay with what you suggest.